mailer/tests.py

```diff
@@ -595,29 +598,29 @@ def test_message_log(self):
 
     def test_message_str(self):
         with self.settings(MAILER_EMAIL_BACKEND="django.core.mail.backends.locmem.EmailBackend"):
-            mailer.send_mail("Subject Msg", "Body", "msg1@example.com", ["rec1@example.com"])
+            mailer.send_mail("Subject Msg 中", "Body 中", "msg1@example.com", ["rec1@example.com"])
 
             self.assertEqual(Message.objects.count(), 1)
 
             msg = Message.objects.get()
             self.assertEqual(
                 str(msg),
-                'On {0}, \"Subject Msg\" to rec1@example.com'.format(msg.when_added),
```

Comment:
Yeah, the decorator allows for `str` to be used. Using `six.text_type` would be working around the problem in the tests.

Try using `six.text_type` in the tests without the `python_2_unicode_compatible` decorator. The tests will be pass but the problem won't be fixed.

The docs explain how the decorator works:

https://docs.djangoproject.com/en/1.8/topics/python3/#str-and-unicode-methods